samples/examples/video.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>mr.js - video</title>
+    <meta name="description" content="mr.js example - video">
+    <script src="/mr.js"></script>
+    <link rel="stylesheet" type="text/css" href="video-style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body >
+   <mr-app>
+    <mr-entity id="rig" data-comp-anchor="type: fixed;">
+        <mr-panel id="tabbar" data-position="0 0 0.001"> <!--TODO: move Z offset to css z-index-->
+            <mr-button onclick="playVideo()" class="col-1">Play</mr-button>
+            <mr-button onclick="stopVideo()" class="col-2">Stop</mr-button>
+            <mr-button onclick="screenShare()" class="col-3">Screenshare</mr-button>
```

Comment:
This is really cool! but probably shouldn't make it into the simple example. Also it doesn't work when in headset. Maybe we replace it with a loop button?